src/test/java/com/digirati/taxonomy/manager/lookup/persistence/SkosTranslatorTest.java

```diff
@@ -0,0 +1,203 @@
+                        .setAltLabel(
+                                "[{\"language\":\"fr\",\"value\":\"troisieme\"},{\"language\":\"en\",\"value\":\"third\"}]");
+        ConceptModel four =
+                new ConceptModel()
+                        .setIri("http://example.com/concept#4")
+                        .setPreferredLabel(
+                                "[{\"language\":\"fr\",\"value\":\"quatre\"},{\"language\":\"en\",\"value\":\"four\"}]")
+                        .setAltLabel(
+                                "[{\"language\":\"fr\",\"value\":\"quatrieme\"},{\"language\":\"en\",\"value\":\"fourth\"}]");
+
+        ConceptSemanticRelationModel oneIsNarrowerThanTwo =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#1")
+                        .setTargetId("http://example.com/concept#2")
+                        .setRelation(SemanticRelationType.NARROWER)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel twoIsRelatedToThree =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#2")
+                        .setTargetId("http://example.com/concept#3")
+                        .setRelation(SemanticRelationType.RELATED)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel twoIsRelatedToFour =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#2")
+                        .setTargetId("http://example.com/concept#4")
+                        .setRelation(SemanticRelationType.RELATED)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel threeIsBroaderThanOne =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#3")
+                        .setTargetId("http://example.com/concept#1")
+                        .setRelation(SemanticRelationType.BROADER)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel threeIsNarrowerThanFour =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#3")
+                        .setTargetId("http://example.com/concept#4")
+                        .setRelation(SemanticRelationType.NARROWER)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel fourIsBroaderThanThree =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#4")
+                        .setTargetId("http://example.com/concept#3")
+                        .setRelation(SemanticRelationType.BROADER)
+                        .setTransitive(false);
+
+        return new RdfModel(
+                Arrays.asList(one, three, two, four),
```

Comment:
What benefit do we get from `List.of` compared to `Arrays.asList`?

Reply:
* It's shorter
* It doesn't coerce variadic argument lists to arrays
* It is immutable and doesn't depend on the underlying array
* It's the replacement API since Java 9
* It's free to use any list implementation (rather than one backed by the underlying coerced array)

This is mentioned in the Java API docs.